main.go

```diff
@@ -239,6 +237,18 @@ func Run(ctx context.Context) int {
 	ctx = bugsnag.StartSession(ctx)
 	defer misc.BugsnagNotify(ctx, "Core")()
 
+	deploymentType, err := deployment.GetFromEnv()
+	if err != nil {
+		pkgLogger.Errorf("failed to get deployment type: %w", err)
+		return 1
+	}
+	// TODO: remove as soon as we update the configuration with statsExcludedTags where necessary
+	if !config.IsSet("statsExcludedTags") && deploymentType == deployment.MultiTenantType && (!config.IsSet("WORKSPACE_NAMESPACE") || strings.Contains(config.GetString("WORKSPACE_NAMESPACE", ""), "free")) {
+		config.Set("statsExcludedTags", []string{"workspaceId"})
+	}
```

Comment:
@chandumlg we should really set this through the config. I think it is only needed in one place, right?

Reply:
Yes.

BTW, do we need !config.IsSet("statsExcludedTags") and !config.IsSet("WORKSPACE_NAMESPACE") checks?

Reply:
` !config.IsSet("statsExcludedTags")` is there to ignore this fragment in case external configuration has been provided for `statsExcludedTags`.

Reply:
Got it.